README.md

```diff
@@ -158,24 +165,29 @@ kubectl.sh get --raw "/apis/kubeprovenance.cloudark.io/v1/namespaces/default/pos
 kubectl.sh get --raw "/apis/kubeprovenance.cloudark.io/v1/namespaces/default/postgreses/client25/spechistory"
 ```
 
-3) Get diff of Postgres custom resource instance between version 1 and version 2
+3) Get diff of Postgres custom resource instance between version 1 and version 5
 
 ```
-kubectl.sh get --raw "/apis/kubeprovenance.cloudark.io/v1/namespaces/default/postgreses/client25/diff?start=1&end=2"
+kubectl.sh get --raw "/apis/kubeprovenance.cloudark.io/v1/namespaces/default/postgreses/client25/diff?start=1&end=5"
 ```
 
-4) Get diff of the field databases for a Postgres custom resource instance between version 1 and version 2
+4) Get diff of the field databases for a Postgres custom resource instance between version 1 and version 5
```

Comment:
This line says version 5 but the query parameter shows version 2.

Can you make this consistent?